### PR TITLE
Add kube-apiserver log option

### PR DESCRIPTION
**Pull request by mnshaw**
\cc @timstclair @rmmh

### CODE REVIEW

File: gubernator/main_test.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ def test_nodelog_missing_kubelet(self):
 
     def test_nodelog_kubelet(self):
         """Test for a kubelet file."""
-        build_dir = self.BUILD_DIR + 'nodelog?pod=abc&junit=junit_01.xml'
+        build_dir = self.BUILD_DIR + 'nodelog?logfile=kubelet.log&pod=abc&junit=junit_01.xml'
```
Comment (rmmh) [inline gubernator/main_test.py]: this shouldn't be called build_dir

File: gubernator/templates/filtered_log.html [hunk 2]
```diff
@@ -8,15 +8,16 @@
 <h1>{% if pr %}<a href="/pr/{{pr}}">PR #{{pr}}</a> {% endif %}<a href="/builds{{job_dir}}">{{job}}</a> <a href="/build{{build_dir}}">#{{build}}</a> Kubelet Log</h1>
 </div>
 <div id="failures">
-% if kubelet_log
-<h3>Lines from kubelet.log containing pod {{pod}}</h3>
+% if log
```
Comment (rmmh) [inline gubernator/templates/filtered_log.html]: this if should probably be down guarding the <pre>

File: gubernator/templates/filtered_log.html [hunk 3]
```diff
@@ -8,16 +8,17 @@
 <h1>{% if pr %}<a href="/pr/{{pr}}">PR #{{pr}}</a> {% endif %}<a href="/builds{{job_dir}}">{{job}}</a> <a href="/build{{build_dir}}">#{{build}}</a> Kubelet Log</h1>
```
Comment (timstclair) [inline gubernator/templates/filtered_log.html]: Should you also update the above mentions of "Kubelet Log"?

**Comment by timstclair** [comment]
Just one question.

**Comment by rmmh** [comment]
LGTM after header fix.

**Pull request merged**